apps/mobile/src/utils/localization/currency.ts

```diff
@@ -0,0 +1,80 @@
+import {bigNumberToString} from '../bigNumberToString'
+
+interface CurrencyInfo {
+    code: string                  // ISO-4217
```

Comment:
It would be good practice to make these properties readonly. 
Also, we are using branded types in the codebase, I would much more prefer defining the interface like this (in currency.brand.ts): 

```
import {z} from 'zod'
export const Currency = z.object({
    code: z.string().brand<'currencyCode'>(), // it can be further refined but that, I don't think is necessary in this case.
    flag: z.string().brand<'currencyFlag'>(),
    name: z.string().brand<'currencyName'>(),
    symbol: z.string().brand<'currencySymbol'>(),
    position: z.enum(['before', 'after']),
})
export type Currency = z.TypeOf<typeof Currency>
```

If we do it that way, and later in the code we write a function that accepts flag. We can force the linter to only accept `Currency['flag']` type instead of generic `string` - since the only place where those brands are created is in this file (when defining `currencies` constant), it is very unlikely we pass value that we did not intend to. This saves a lot of headache later on ;).